lib/rucio/client/baseclient.py

```diff
@@ -279,6 +271,19 @@ def __init__(self, rucio_host=None, auth_host=None, account=None, ca_cert=None,
                 except (NoOptionError, NoSectionError):
                     self.vo = 'def'
 
+        # if token file path is defined in the rucio.cfg file, use that file. Currently this prevents authenticating as another user or VO.
+        if self.auth_token_file_path:
+            self.token_file = self.auth_token_file_path
+            self.token_path = '/'.join(self.token_file.split('/')[:-1])
+            self.token_exp_epoch_file = self.token_path + '/' + self.TOKEN_EXP_PREFIX + self.account
+        else:
+            self.token_path = self.TOKEN_PATH_PREFIX + self.account
+            self.token_file = self.token_path + '/' + self.TOKEN_PREFIX + self.account
+            self.token_exp_epoch_file = self.token_path + '/' + self.TOKEN_EXP_PREFIX + self.account
+            if self.vo != 'def':
+                self.token_file += '@%s' % self.vo
+                self.token_exp_epoch_file += '@%s' % self.vo
+
```

Comment:
Am I right in thinking that we are keeping this change as it is after that discussion with Martin?